ba/tutorials/tour/annotations.md

```diff
@@ -0,0 +1,132 @@
+|  [`scala.reflect.BeanProperty`](http://www.scala-lang.org/api/2.9.1/scala/reflect/BeanProperty.html) |  [`Design pattern`](http://docs.oracle.com/javase/tutorial/javabeans/writing/properties.html) |
+
+U sljedećem primjeru dodajemo `throws` anotaciju definiciji metode `read` da bi uhvatili izuzetak(exception) u Java main programu.
+
+> Java kompajler provjerava da li program sadrži rukovatelje(handler) za [provjereni izuzetak(checked exception)](http://docs.oracle.com/javase/specs/jls/se5.0/html/exceptions.html) 
+analizom koji se izuzeci mogu dobiti izvršenjem neke metode ili konstruktora.
+Za svaki mogući provjereni izuzetak **throws** klauza metodi ili konstruktora _mora_ navesti klasu izuzetka ili neku nadklasu izuzetka.
+> Pošto Scala nema provjerene izuzetke, Scala metode _moraju_ imati jednu ili više `throws` anotacija kako bi Java kod mogao uhvatiti iste.
+
+    package examples
+    import java.io._
+    class Reader(fname: String) {
+      private val in = new BufferedReader(new FileReader(fname))
+      @throws(classOf[IOException])
+      def read() = in.read()
+    }
+
+Sljedeći Java program prikazuje sadržaj fajla s imenom koje je proslijeđeno kao prvi argument `main` metode.
+
+    package test;
+    import examples.Reader;  // Scala class !!
+    public class AnnotaTest {
+        public static void main(String[] args) {
+            try {
+                Reader in = new Reader(args[0]);
+                int c;
+                while ((c = in.read()) != -1) {
+                    System.out.print((char) c);
+                }
+            } catch (java.io.IOException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+    }
+
+Kada bi zakomentarisali `throws` anotaciju u klasi Reader dobili bi sljedeću grešku s porukom pri kompajliranju Java main programa:
+
+    Main.java:11: exception java.io.IOException is never thrown in body of
+    corresponding try statement
+            } catch (java.io.IOException e) {
+              ^
+    1 error
+
+### Java anotacije ###
+
+**Napomena:** Pobrinite se da koristite `-target:jvm-1.5` opciju sa Java anotacijama.
+
+Java 1.5 je uvela korisnički definisane metapodatke u formi [anotacija](http://java.sun.com/j2se/1.5.0/docs/guide/language/annotations.html). Ključna sposobnost anotacija je da koriste parove ime-vrijednost za inicijalizaciju svojih elemenata. Naprimjer, ako nam treba anotacija da pratimo izvor neke klase mogli bismo je definisati kao
+
+    @interface Source {
+      public String URL();
+      public String mail();
+    }
+
+I upotrijebiti je kao npr:
+
+    @Source(URL = "http://coders.com/",
+            mail = "support@coders.com")
+    public class MyClass extends HisClass ...
+
+Primjena anotacije u Scali izgleda kao poziv konstruktora, dok se za instanciranje Javinih anotacija moraju koristiti imenovani argumenti:
```

Comment:
I left this one as before, but if you insist I'll change it. The thing is that there are some other words that should be treated same, like "Java" for example. I'd then have to change it also, like "Java-i", right?